snow/engine/snowman/transitive_test.go

```diff
@@ -343,7 +337,7 @@ func TestEngineQuery(t *testing.T) {
 			t.Fatalf("Asking for wrong block")
 		}
 	}
-	if err := te.Chits(context.Background(), vdr, *queryRequestID, []ids.ID{blk1.ID()}, nil); err != nil {
+	if err := te.Chits(context.Background(), vdr, *queryRequestID, blk1.ID(), blk1.ID()); err != nil {
```

Comment:
why did we go from passing `nil` to accepted to passing the block ID?

Reply:
This change requires the second field to be populated